packages/client/tests/functional/issues/14954-date-batch/tests.ts

```diff
@@ -0,0 +1,112 @@
+import testMatrix from './_matrix'
+// @ts-ignore
+import type { PrismaClient } from './node_modules/@prisma/client'
+
+declare let prisma: PrismaClient
+
+testMatrix.setupTestSuite(() => {
```

Comment:
I would even suggest that these dates are always generated so we know we are testing a random date each time: 

```js
const date1 = new Date().toString()
const date2 = new Date().toString();
```

Reply:
const date1 = new Date().toISOString()

I remember I could not use that one, because that actually worked. The queries only failed when the date had the `.000Z` omitted.